Dockerfile

```diff
@@ -19,9 +19,6 @@ RUN  cd /usr/share/heartbeat/.node \\
       && mkdir node \\
       && curl https://nodejs.org/dist/v12.18.4/node-v12.18.4-linux-x64.tar.xz | tar -xJ --strip 1 -C node
 ENV PATH="/usr/share/heartbeat/.node/node/bin:$PATH"
-# Install playwright first since it speeds up the install of elastic-synthetics*.tgz, since it doesn't need to re-download the
-# browsers every time the code there changes
-RUN npm i -g playwright-chromium
-COPY elastic-synthetics-*.tgz /opt/elastic-synthetics.tgz
-RUN npm install -g /opt/elastic-synthetics.tgz
-ENV HEARTBEAT_SYNTHETICS_TGZ=/opt/elastic-synthetics.tgz
+# Install the latest version of @elastic/synthetics, so heartbeat can
+# call the executable directly
+RUN npm i -g @elastic/synthetics
```

Comment:
Once we publish the new version, this should make `elastic-synthetics` in the docker image work out of the box.